assets/js/data/collections/selectors.js

```diff
@@ -7,31 +7,110 @@ import { addQueryArgs } from '@wordpress/url';
  * Internal dependencies
  */
 import { hasInState } from '../utils';
+import { DEFAULT_EMPTY_ARRAY } from './constants';
 
-const DEFAULT_EMPTY_ARRAY = [];
+const getFromState = (
+	state,
+	namespace,
+	modelName,
+	query,
+	ids,
```

Comment:
In the arguments list `ids` appears after `query` but when we retrieve the state, `query` appears after `ids` (_state[ namespace ][ modelName ]**[ ids ][ query ]**[ type ]_). I find it a bit confusing. Could we change the arguments order to match the state shape? Or make `getFromState` accept an object instead of a list of arguments so order doesn't matter?

Reply:
The argument list is in order of likelihood to be used (where query will likely be used more than ids).  But ya it's a good observation, I like the idea of making `getFromState` accept an object. I'll give that a go.